src/app/shared/pipes/duration/duration.pipe.ts

```diff
@@ -0,0 +1,19 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'duration'
+})
+export class DurationPipe implements PipeTransform {
+
+  transform(value: number): unknown {
+    const date = new Date(0);
+    date.setMinutes(0);
+    date.setHours(0);
+    date.setMinutes(value);
+    const hours = date.getHours();
+    let minutes = `${date.getMinutes()}`;
+    if (Number(minutes) < 10) minutes = '0' + minutes;
```

Comment:
it is better to create helper, which add 0 to beginning. Create function `padZero` or smth like this

Reply:
fixed